CMakeLists.txt

```
@@ -36,6 +36,13 @@ option(BUILD_SHARED_LIBS "Build the shared library (instead of a static lib)" TR
 option(NEW_HOST_WRAPPER "Use the new host wrapper" TRUE)
 option(SF2_SUPPORT "Enable SF2 support and float DCF" TRUE)
 option(ZLIB_SUPPORT "Enable XMF ZLIB Unpacker support" TRUE)
+option(EAS_WT_SYNTH "Enable WaveTable Synth and build its sound library" TRUE)
+option(EAS_FM_SYNTH "Enable FM Synth and build its sound library" FALSE)
+option(EAS_HYBRID_SYNTH "Enable Hybrid Synth and build its sound library" FALSE)
```

I would move this option below, and convert it into `cmake_dependent_option`:

```suggestion
cmake_dependent_option(EAS_HYBRID_SYNTH "Enable Hybrid Synth" FALSE "NOT USE_44KHZ;NOT USE_16BITS_SAMPLES" FALSE)
```